examples/drivers/windows/otLwf/command.c

```diff
@@ -127,6 +134,12 @@ otLwfCmdInitialize(
             break;
         }
 
+        NtStatus = otLwfCmdResetDevice(pFilter, FALSE);
+        if (!NT_SUCCESS(NtStatus))
+        {
+            Status = NDIS_STATUS_FAILURE;
+        }
+
     } while (FALSE);
```

Comment:
Why is this in a ``while (FALSE)`` loop?  Isn't that just the same as making it a code block?  It will never actualy loop as far as I can see.